great_expectations/cli/suite.py

```diff
@@ -94,20 +103,77 @@ def suite(ctx):
     help="By default launch jupyter notebooks, unless you specify --no-jupyter flag.",
 )
 @click.pass_context
-def suite_new(ctx, expectation_suite, interactive, profile, batch_request, no_jupyter):
+def suite_new(
+    ctx,
+    expectation_suite,
+    interactive_flag,
+    no_interactive_flag,
+    profile,
+    batch_request,
+    no_jupyter,
+):
     """
     Create a new empty Expectation Suite.
     Edit in jupyter notebooks, or skip with the --no-jupyter flag.
     """
     context: DataContext = ctx.obj.data_context
     usage_event_end: str = ctx.obj.usage_event_end
 
+    processed_flags = _process_suite_new_flags_and_prompt(
```

Comment:
@anthonyburdi How about adding a type hint -- we have it everywhere in SUITE and CHECKPOINT (and most other places)... Thanks!

Reply:
Done! Thanks for the reminder.